Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -46,14 +46,31 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         },
 
         setUp: function () {
-            this.model = new Y.eZ.Location();
+            this.locationId = '1/2/3';
+            this.model = new Y.eZ.Location({id: this.locationId});
+            this.contentServiceMock = new Y.Mock();
         },
 
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "Should have a move method": function () {
```

Comment:
bad name, the location model should indeed have a move method but the important thing is that it moves the location... `"Should move the location"` would be better and you should put this test in its own test case dedicated to the move method.